js/src/components/ui/footer/Footer.tsx

```diff
@@ -40,6 +40,15 @@ export function Footer() {
           >
             <FaGithub size={24} />
           </FooterIconLink>
+          <Anchor
+            href="/privacy"
+            c="dimmed"
+            size="sm"
+            variant="subtle"
+            underline="always"
+          >
```

Comment:
The Privacy Policy link seems to be misaligned. We can add a `margin-top` (or `mt` prop in Mantine) to help balance this out for now. `xs` seems to be too large, so we can just add a slightly-hacky solution of `pt={5}` for now.

```suggestion
            underline="always"
            /* need to fix centering issue, mt={"sm"} is too large */
            mt={5}
          >
```